### DISCUSSION TITLE
Custom Snippets?

### DISCUSSION THREAD

**Discussion by WizardOfOz-1** [saghen/blink.cmp#364] [Q&A] [answered]
Blink CMP supports `vim.snippet` and `friendly-snippets.` ,  is there a way to add custom snippets by pointing to a custom folder? I couldn't find anything in the documentation, am I missing something?

**Comment by saghen** [comment, marked as answer]
By default, blink.cmp will search `$yourConfig/snippets` (i.e. `~/.config/nvim/snippets`) in the same way [nvim-snippets](https://github.com/garymjr/nvim-snippets) does. You may add additional paths to search via `sources.providers.snippets.opts.search_paths`

**Reply by mmirus** [reply]
One thing you can do using this option is specify the location of addition snippet collections you have added as dependencies. Here's an example using lazy.nvim which tells blink to load the karuna/vscode-rspec-snippets snippets from the location to which lazy installs them:

```lua
return {
  "saghen/blink.cmp",
  dependencies = { "karuna/vscode-rspec-snippets" },
  opts = {
    -- ...
    sources = {
      -- ...
      providers = {
        -- ...
        snippets = {
          opts = {
            search_paths = {
              vim.fn.stdpath("config") .. "/snippets",
              vim.fn.stdpath("data") .. "/lazy/" .. "vscode-rspec-snippets",
            },
          },
        },
      },
    },
  -- ...
```
